proposals/records.md

```diff
@@ -1,75 +1,50 @@
-# Records Work-in-Progress
 
-Unlike the other records proposals, this is not a proposal in itself, but a work-in-progress designed to record consensus design
-decisions for the records feature. Specification detail will be added as necessary to resolve questions.
+# Records
 
-The syntax for a record is proposed to be added as follows:
+This proposal tracks the specification for the C# 9 records feature, as agreed to by the C#
+language design team.
 
-```antlr
-class_declaration
-    : attributes? class_modifiers? 'partial'? 'class' identifier type_parameter_list?
-      parameter_list? type_parameter_constraints_clauses? class_body
-    ;
+The syntax for a record is as follows:
 
-struct_declaration
-    : attributes? struct_modifiers? 'partial'? 'struct' identifier type_parameter_list?
-      parameter_list? struct_interfaces? type_parameter_constraints_clauses? struct_body
+```antlr
+record_declaration
+    : attributes? class_modifier* 'partial'? 'record' identifier type_parameter_list?
+      parameter_list? record_base? type_parameter_constraints_clause* record_body
     ;
 
-class_body
-    : '{' class_member_declarations? '}'
-    | ';'
+record_base
+    : ':' class_type argument_list?
```

Comment:
>argument_list [](start = 21, length = 13)

The spec doesn't explain the meaning of the `argument_list` for a record declaration without a `parameter_list`.

Reply:
> The spec doesn't explain the meaning of the argument_list for a record declaration without a parameter_list.

@agocke I think we should make it clear that arguments are not allowed unless there is a parameter_list